tests/cases/conformance/types/mapped/mappedTypeErrors.ts

```diff
@@ -0,0 +1,62 @@
+// @strictNullChecks: true
+// @declaration: true
+
+type Partial<T> = {

```

Comment:
Will fix that in a separate pull request for introducing the special mapped types in lib.d.ts.